pypeit/pypeit.py

```diff
@@ -391,6 +418,21 @@ def helio_correct(self, sobjs, maskslits, frame, obstime):
         return vel_corr
 
     def get_sci_metadata(self, frame, det):
+        """
+        Grab the meta data for a given science frame and specific detector
+
+        Args:
+            frame (int): Frame index
+            det (int): Detector index
+
+        Returns:
```

Comment:
The sphinx representation of this won't respect the enumeration.  But you could use a list.  E.g.:
```
Returns:
    Five objects are returned::
        - str: Object type; 'science' or 'standard'
        - str: Setup
....
```

Reply:
done